mixer/cmd/bundles.go

```diff
@@ -198,12 +201,74 @@ any errors are encountered earlier on.`,
 	},
 }
 
+// Bundle validate command ('mixer bundle validate')
+type bundleValidateCmdFlags struct {
+	allLocal      bool
+	strictnessLvl string
+}
+
+var bundleValidateFlags bundleValidateCmdFlags
+
+var bundleValidateCmd = &cobra.Command{
+	Use:   "validate <bundle> [<bundle>...]",
+	Short: "Validate local bundle definition files",
+	Long: `Checks local bundle definition files for validity as a separate step. While
+'mixer bundle edit' will automatically validate bundles as you edit them, it can
+be useful to manually run this validation step on bundles you have previously
+written, or have edited using a different tool. Only local bundle files are
+checked; upstream bundles are trusted as valid. Valid bundles yield no output.
+Any invalid bundles will yield a non-zero return code.
+
+An optional '--strictness-level' flag allows you to control how strictly bundles
```

Comment:
Optional: which strictness version does edit use?

Reply:
Fixed in new push.